backend/question-service/src/controllers/handlers/get-handler.ts

```diff
@@ -1,42 +1,60 @@
 import { Request, Response } from "express";
 import HttpStatusCode from "../../lib/enums/HttpStatusCode";
 import { QueryParamValidator } from "../../lib/validators/QueryParamValidator";
-import questionDb from "../../models/database/schema/question";
-import mongoose from "mongoose";
+import db from "../../models/db";
 import { ZodError } from "zod";
 import { formatErrorMessage } from "../../lib/utils/errorUtils";
 import Topic from "../../lib/enums/Topic";
+import Complexity from "../../lib/enums/Complexity";
+import { Example } from "../../models/question";
 
 // Check if database connection is successful
 export const getHealth = async (_: Request, response: Response) => {
-  if (mongoose.connection.readyState === 1) {
+  try {
+    const result = await db.$queryRaw`SELECT 1`;
+
+    if (!result) {
+      throw new Error("No database connection from the server");
+    }
+
     response.status(HttpStatusCode.OK).json({ message: "Healthy" });
-  } else {
+  } catch (error) {
+    console.log(error);
     response.status(HttpStatusCode.INTERNAL_SERVER_ERROR).json({
-      message: "Database connection is not established.",
+      error: "INTERNAL SERVER ERROR",
+      message: "No database connection from the server",
     });
   }
 };
 
+type Filter = {
+  topics?: string[];
+  difficulties?: string[];
+  author?: string;
+};
+
 // Query mulitple questions from database by filter
 export const getQuestions = async (request: Request, response: Response) => {
   try {
     // check if there are any query params provided and the provided values are valid
-    const { topics, complexity, author } = QueryParamValidator.parse(
+    const { topics, difficulties, author } = QueryParamValidator.parse(
```

Comment:
qn! is there a reason u switched from complexity to difficulty? just curious, difficulties sounds ok

Reply:
Emm, the reason is because I was trying to avoid naming as `complexities` as the query param accepts multiple complexities haha...